src/style.css

```diff
@@ -0,0 +1,90 @@
+/* fonts */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  box-sizing: border-box;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+input.search-big,
+input.search-small {
+  width: 100%;
+  box-sizing: border-box;
+  display: block;
+}
+
+/* big */
+.search-big {
+  height: 70px;
+  margin-bottom: 30px;
+  padding-left: 80px;
+
+  font-size: 16px;
+  line-height: 70px;
+
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 0 5px 1px #e1e7ed;
+  background: url(./images/Search.svg) no-repeat scroll 30px 25px;
+}
+
+.search-big:hover {
+  box-shadow: 3px 3px 5px 1px #e1e7ed;
+}
+
+.search-big:focus {
+  font-weight: bold;
+
+  box-shadow: inset 0 -40px 35px 15px rgba(248, 248, 249, 0.6);
+  border: 1px solid #e1e7ed;
+  outline: none;
+}
+
+.search-big::placeholder {
+  font-weight: normal;
+  color: #3d4e61;
+}
+
+/* small */
+.search-small {
+  height: 42px;
```

Comment:
You are repeating too many styles in your `search-big` and `search-small` classes. You can create one common class `search` and give at all common classes for both inputs and use it then like `class="search search-big" e.g